classes/User.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+from utilities.hash_utilities import generate_hash
+from utilities.db_util import connect_execute_query
+
+
+class User:
+    def __init__(self,name,password):
+        self.name=name;
+        self.password=password
+    
+    
+    def validate_user(self) -> bool:
+        query=f"SELECT user FROM users WHERE user=\'{self.name}\'"
+        result=connect_execute_query(query)
+        if result:
+            logging.warning(f"User by the name {result[0][0]} already exists.")
+            return False
+        else:
+            logging.info(f"User with name {self.name} is valid.")
+            return True
+
+
+    def input_user(self) -> bool:
+        if self.validate_user():
+            logging.info("Adding user into db.")
+            query=f"INSERT INTO users (user, password) VALUES {self.name, generate_hash(self.password)}"
+            result=connect_execute_query(query)
+            return True
+        else:
+            return False
```

Comment:
1. It would be nice to have log on which user is being added etc. `f"Adding user [{self.user}] into DB"`
2. We already have template query for inserting rows in user table - `db_templates/user_insert.template` and path to it in `defaults.py` file, we should use templates as much as we can
3. Minor one - you can just say `return False` without else statement, or even something like
```
if not self.validate_user():
   return False

logging.info(f"Adding user [self.user] into DB.")
# load template
connect_execute_query(query)
return True
```
It's a good practice that requirements for logic are checked on top of the method